chirp/wxui/main.py

```diff
@@ -1835,7 +1836,12 @@ def _do_network_query(self, query_cls):
             self.add_editorset(editorset)
 
     def _menu_query_rr(self, event):
-        self._do_network_query(query_sources.RRQueryDialog)
+        if radioreference.HAVE_SUDS:
+            self._do_network_query(query_sources.RRQueryDialog)
+        else:
+            common.error_proof.show_error(
+                _('The python3-suds package is required '
```

Comment:
Perhaps you inferred that Linux was the problem child because this package isn't listed in the set of distro packages on the linux page. However, linux users are getting that via pip/pipx currently.

Especially with the "managed environment" flag in modern distros, we might as well just install everything we can from pip instead of depending on the problems that come from varying distro version support for things (wxpython being the exception that we really need to get from the distro).